src/backlinks/handler.js

```diff
@@ -143,6 +133,24 @@ export default async function auditBrokenBacklinks(message, context) {
       auditContext,
       auditResult,
     };
+
+    try {
```

Comment:
I would push it even further down after `sqs.sendMessage`, as it would be a sort of post-processor (https://github.com/adobe/spacecat-audit-worker/blob/main/src/common/audit.js#L123-L126)